### PR TITLE
Allow everyone to unstake YY and unwind LP tokens when insolvent

### PR DESCRIPTION
**Pull request by mbare0**

### CODE REVIEW

File: contracts/facets/UniswapV2DEXFacet.sol [hunk 1]
```diff
@@ -83,7 +84,7 @@ contract UniswapV2DEXFacet is ReentrancyGuardKeccak, SolvencyMethods {
     /**
     * Adds liquidity
     **/
-    function addLiquidity(bytes32 _assetA, bytes32 _assetB, uint amountA, uint amountB, uint amountAMin, uint amountBMin) internal recalculateAssetsExposure remainsSolvent {
+    function addLiquidity(bytes32 _assetA, bytes32 _assetB, uint amountA, uint amountB, uint amountAMin, uint amountBMin) internal recalculateAssetsExposure onlyOwnerOrInsolvent {
```
Comment (kavsky02) [inline contracts/facets/UniswapV2DEXFacet.sol:87]: Do we need to add liquidity if insolvent?

File: contracts/facets/UniswapV2DEXFacet.sol [hunk 2]
```diff
@@ -8,12 +8,13 @@ import "@uniswap/lib/contracts/libraries/TransferHelper.sol";
 import "@openzeppelin/contracts/utils/math/Math.sol";
 import "../lib/SolvencyMethods.sol";
 import {DiamondStorageLib} from "../lib/DiamondStorageLib.sol";
+import "../OnlyOwnerOrInsolvent.sol";
```
Comment (kavsky02) [inline contracts/facets/UniswapV2DEXFacet.sol:11]: Can we maybe rename this modifier to `OnlyOwnerRemainSolventOrInsolvent?
The name is a bit long but better describes what this modifier actually does

File: contracts/facets/UniswapV2DEXFacet.sol (hunk 1 above)
Comment (mbare0) [reply contracts/facets/UniswapV2DEXFacet.sol:87]: no

File: contracts/facets/UniswapV2DEXFacet.sol (hunk 2 above)
Comment (mbare0) [reply contracts/facets/UniswapV2DEXFacet.sol:11]: after launch plox xD

Comment (kavsky02) [reply contracts/facets/UniswapV2DEXFacet.sol:11]: xD

**Review by kavsky02 — APPROVED** [review]

**Pull request merged**